app/src/org/commcare/activities/CommCareActivity.java

```diff
@@ -635,7 +635,7 @@ protected void tryToAddActionSearchBar(Activity act, Menu menu,
                                            ActionBarInstantiator instantiator) {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.HONEYCOMB) {
             MenuInflater inflater = act.getMenuInflater();
-            inflater.inflate(org.commcare.dalvik.R.menu.activity_report_problem, menu);
+            inflater.inflate(org.commcare.dalvik.R.menu.action_bar_search_view, menu);

```

Comment:
Yea sorry I saw that but that's what I'm confused about-- I don't understand how you just renamed one to the other, since they don't sound related at all.